api/src/event/entities/event.entity.ts

```diff
@@ -24,15 +24,20 @@ export abstract class EventEntity {
   @Column()
   field: string;
 
+  @Column({ nullable: true })
+  subfield: string;
+
   @Column()
   previous: string;
 
   @Column()
   updated: string;
 
+  @Column({ nullable: true })
+  language: string;
+
```

Comment:
looks like you add new fields, then you might have to add new migration to add those new columns.

Reply:
oh yeah good point! made a mental note but didn't follow up